Makefile

```diff
@@ -575,6 +577,37 @@ edpm_deploy: input edpm_deploy_prep ## installs the dataplane instance using kus
 	$(eval $(call vars,$@,dataplane))
 	oc kustomize ${DEPLOY_DIR} | oc apply -f -
 
+.PHONY: edpm_deploy_baremetal_prep
+edpm_deploy_baremetal_prep: export KIND=OpenStackDataPlane
+edpm_deploy_baremetal_prep: export EDPM_ANSIBLE_SECRET=${DATAPLANE_ANSIBLE_SECRET}
+edpm_deploy_baremetal_prep: export EDPM_ANSIBLE_USER=cloud-amin
+edpm_deploy_baremetal_prep: export BMH_NAMESPACE=openstack
```

Comment:
The `openstack` here needs to be replaced with something dynamic, I think

Reply:
yeah we can use ${NAMESPACE}, but it's something that has to be provided in most cases.

Reply:
Would that work if you wanted to deploy the `OSDP` in `openstack` namespace, but have BMHs in `openshift-machine-api`?  That's the dilemma we face against an AI/IPI OCP cluster.  But maybe I could have created my BMHs in `openstack` instead of `openshift-machine-api`.  It just used to be that you had to put them in the same namespace as the Baremetal operator, which happens to be `openshift-machine-api` in AI/IPI installations.

Reply:
the default is `openshift-machine-api`, If your BMH is in `openshift-machine-api`  then you've to set `$BMH_NAMESPACE=openshift-machine-api` with the way we've written the kustomization for this target.